File: src/Lista02/Questao04/dados/Aluno.java
package Lista02.Questao04.dados;

public class Aluno {
    private String nome;
    private int idade;
    private double[] notas = new double[5];

    public String getNome() {
        return nome;
    }

    public void setNome(String nome) {
        this.nome = nome;
    }

    public int getIdade() {
        return idade;
    }

    public void setIdade(int idade) {
        this.idade = idade;
    }

    public double[] getNotas() {
        return notas;
    }

    public void setNotas(double[] notas) {
        this.notas = notas;
    }

    public double calcularMedia() {
        double media = 0;
        for (double nota : notas) {
            media += nota;
        }
        media /= notas.length;

        return media;
    }

    public String toString() {
        return "Nome: " + this.nome + ", Idade: " + this.idade + ", Média: " + calcularMedia();
    }
}

File: src/Lista03/Questao04/Aluno.py
from Pessoa import Pessoa

class Aluno(Pessoa):
    def __init__(self, nome, notas):
        self.__nome = nome
        self.__notas = notas

    def calcularMedia(self):
        somatorio = 0

        for nota in self.__notas:
            somatorio += nota

        return somatorio / len(self.__notas)

    def __repr__(self):
        s = "Nome: " + self.__nome + "\n"
        for i in range(len(self.__notas)):
            s += "Nota " + str(i+1) + ": " + str("{:.2f}").format(self.__notas[i])
            s += "\n"
        s += "Media: " + str("{:.2f}").format(self.calcularMedia())
        s += '\n'
        if self.calcularMedia() >= 7:
            s += "Aprovado"
        else:
            s += "Reprovado"

        return s